po/bn.po

```diff
@@ -251,431 +251,431 @@ msgstr "দিন 2: সকাল"
-msgstr ""
+msgstr "Option and Result"
 
 #: src/SUMMARY.md:108
 msgid "String"
-msgstr ""
+msgstr "String"
 
 #: src/SUMMARY.md:109
 msgid "Vec"
-msgstr ""
+msgstr "Vec"
 
 #: src/SUMMARY.md:110
 msgid "HashMap"
-msgstr ""
+msgstr "HashMap"
 
 #: src/SUMMARY.md:111
 msgid "Box"
-msgstr ""
+msgstr "Box"
 
 #: src/SUMMARY.md:112
 msgid "Recursive Data Types"
-msgstr ""
+msgstr "Recursive Data ধরন"
 
 #: src/SUMMARY.md:113
 msgid "Niche Optimization"
-msgstr ""
+msgstr "Niche Optimization"
 
 #: src/SUMMARY.md:114
 msgid "Rc"
-msgstr ""
+msgstr "Rc"
 
 #: src/SUMMARY.md:115
 msgid "Modules"
-msgstr ""
+msgstr "Modules"
 
 #: src/SUMMARY.md:116
 msgid "Visibility"
-msgstr ""
+msgstr "দৃশ্যমানতা"
 
 #: src/SUMMARY.md:117
 msgid "Paths"
-msgstr ""
+msgstr "Paths"
 
 #: src/SUMMARY.md:118
 msgid "Filesystem Hierarchy"
-msgstr ""
+msgstr "Filesystem Hierarchy"
 
 #: src/SUMMARY.md:120
 msgid "Luhn Algorithm"
-msgstr ""
+msgstr "Luhn অ্যালগরিদম"
 
 #: src/SUMMARY.md:121
 msgid "Strings and Iterators"
-msgstr ""
+msgstr "Strings and Iterators"
 
 #: src/SUMMARY.md:124
 msgid "Day 3: Morning"
-msgstr ""
+msgstr "তৃতীয় দিনঃ সকাল"
 
 #: src/SUMMARY.md:129
 msgid "Generics"
-msgstr ""
+msgstr "Generics"
 
 #: src/SUMMARY.md:130
 msgid "Generic Data Types"
-msgstr ""
+msgstr "সাধারণ ডেটা ধরণ"
 
 #: src/SUMMARY.md:131
 msgid "Generic Methods"
-msgstr ""
+msgstr "সাধারণ পদ্ধতিগুলো"
 
 #: src/SUMMARY.md:132
 msgid "Monomorphization"
-msgstr ""
+msgstr "Monomorphization"
 
 #: src/SUMMARY.md:133
 msgid "Traits"
-msgstr ""
+msgstr "Traits"
 
 #: src/SUMMARY.md:134
 msgid "Trait Objects"
-msgstr ""
+msgstr "Trait Objects"
 
 #: src/SUMMARY.md:135
 msgid "Deriving Traits"
-msgstr ""
+msgstr "Deriving Traits"
 
 #: src/SUMMARY.md:136
 msgid "Default Methods"
-msgstr ""
+msgstr "ডিফল্ট পদ্ধতি"
 
 #: src/SUMMARY.md:137
 msgid "Trait Bounds"
-msgstr ""
+msgstr "Trait Bounds"
 
 #: src/SUMMARY.md:138
 msgid "impl Trait"
-msgstr ""
+msgstr "impl Trait"
 
 #: src/SUMMARY.md:139
 msgid "Important Traits"
-msgstr ""
+msgstr "গুরুত্বপূর্ণ Traits"
 
 #: src/SUMMARY.md:140
 msgid "Iterator"
-msgstr ""
+msgstr "Iterator"
 
 #: src/SUMMARY.md:141
 msgid "FromIterator"
-msgstr ""
+msgstr "FromIterator"
 
 #: src/SUMMARY.md:142
 msgid "From and Into"
-msgstr ""
+msgstr "From and Into"
 
 #: src/SUMMARY.md:143
 msgid "Read and Write"
-msgstr ""
+msgstr "Read and Write"
 
 #: src/SUMMARY.md:144
 msgid "Drop"
-msgstr ""
+msgstr "Drop"
 
 #: src/SUMMARY.md:145
 msgid "Default"
-msgstr ""
+msgstr "ডিফল্ট"
 
 #: src/SUMMARY.md:146
 msgid "Operators: Add, Mul, ..."
-msgstr ""
+msgstr "অপারেটরঃ Add, Mul, ..."
 
 #: src/SUMMARY.md:147
 msgid "Closures: Fn, FnMut, FnOnce"
-msgstr ""
+msgstr "Closures: Fn, FnMut, FnOnce"
 
 #: src/SUMMARY.md:149
 msgid "A Simple GUI Library"
-msgstr ""
+msgstr "একটি সাধারণ GUI  লাইব্রেরি"
 
 #: src/SUMMARY.md:151
 msgid "Day 3: Afternoon"
-msgstr ""
+msgstr "তৃতীয় দিনঃ বিকাল"
 
 #: src/SUMMARY.md:153
 msgid "Error Handling"
-msgstr ""
+msgstr "ত্রুটি পরিচালনা"
```

Comment:
পরিচালনা -> সামলানো?

Reply:
Hey @mirwasi, Let me just jump in and introduce the GitHub suggestion feature — it's the leftmost button above.

If I understand correctly, you want to suggest this change:

```suggestion
msgstr "ত্রুটি সামলানো"
```

@raselmandol (or yourself) can commit this with a single click afterwards. It can save a lot of round-trips 🙂